### PR TITLE
Adapt the dropzone on the design of the product page

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Look at my dropzone. My dropzone is beautiful. |
| Type? | improvement |
| Category? | BO |
| BC breaks? | Nope |
| Deprecations? | Nope |
| Fixed ticket? | [BOOM-638](http://forge.prestashop.com/browse/BOOM-638) & [BOOM-847](http://forge.prestashop.com/browse/BOOM-847) |
| How to test? | Everything to review is in the Jira issue. <ul><li>Check the icon fits the screenshot</li><li>Only the + mark can now open the file manager</li><li>The image form has changed (with animations, wow)</li><li>The image previews are now smooth</li><li>No dashed border all around empty dropzone</li><li>Progress bar upload & image product form now fit the mock up</li> |

</ul>

### CODE REVIEW

**Comment by Quetzacoalt91** [comment]
capture du 2016-07-01 17-06-47

**Comment by maximebiloe** [comment]
ozone

Dropzone? 🎶  Ma-ia-hii, Ma-ia-huu 🎶

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 1]
```diff
@@ -1233,19 +1245,20 @@ var imagesProduct = (function() {
         paramName: 'form[file]',
         maxFilesize: dropZoneElem.attr('data-max-size'),
         addRemoveLinks: true,
-        clickable: true,
-        thumbnailWidth: 130,
+        clickable: ".openfilemanager",
```
Comment (fojtcz) [inline admin-dev/themes/default/js/bundle/product/form.js]: single quote 😄

**Comment by vincentbz** [comment]
Got one !
If I delete the images, I have the two borders, there should be only the dotted one

image

**Comment by Quetzacoalt91** [comment]
PR updated:
* Removed dashed border all around empty dropzone
* Updated image product form (action buttons)
* Fixed [BOOM-847](http://forge.prestashop.com/browse/BOOM-847)
* Set proper style on progress bar

**Comment by dSkrbic** [comment]
Tests for BOOM-847 **OK**

See added screenshots
z0-screenshot_dsk_2016 07 11_10h14m24s_043_
z1-screenshot_dsk_2016 07 11_10h12m30s_042_

**Comment by Shudrum** [comment]
Thank you 👍 

Be careful of the lines length next time please :)

**Comment by Quetzacoalt91** [comment]
Yes sir !

**Comment by vincentbz** [comment]
happy2

**Pull request merged**